dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -209,9 +209,9 @@
 
             <StaticResource x:Key="NavigationViewItemSeparatorForeground" ResourceKey="SystemControlForegroundBaseLowBrush" />
 
-            <StaticResource x:Key="NavigationViewSelectionIndicatorForeground" ResourceKey="SystemColorHighlightTextColor" />
-            
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="NavigationViewItemForeground" />
+            <SolidColorBrush x:Key="NavigationViewSelectionIndicatorForeground" Color="{ThemeResource SystemColorHighlightTextColor}" />
```

Comment:
why is ThemeResource needed here?

Reply:
@karenbtlai there's a framework issue (see [#2958](https://github.com/microsoft/microsoft-ui-xaml/issues/2958) comments), that prevents from using proper colors when switching between White and Dark HC. In order to resolve that, I found no other way as to use ThemeResource here :)